src/modules/room.js

```diff
@@ -162,22 +169,63 @@ class Room {
         }
     }
 
-    listenToRoomEvents(room){
-        room.on('peer joined', peer => {
+    getPeers() {
+        return [...this.peers]; // Readonly copy
+    }
+
+    pollPeers(intervalMs = 1000) {
```

Comment:
@fazo96 Oh man! Just checked ipfs-pubsub-room does polling for peers too! But I think it is just polling the libp2p peer list. Just adding a note.

Reply:
Yeah, but pubsub-room did other things that would have broken ipfs-api compatibility, so we had to drop it, but then after I reimplemented the polling of the peers here the guys behind orbit-db made this: https://github.com/haadcode/ipfs-pubsub-peer-monitor

So yeah bad timing making the switch

Reply:
Doesn't matter. I think their monitor is still polling anyway.

Reply:
Yeah, I checked and it's implemented the same way I did it